playwright/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:focal
```

Comment:
This is playwrights recommended docker setup. This image allows us to use Chromium, WebKit, and Firefox for tests. Below is a Debian based Dockerfiles that keeps us closer to what we're using for other images in the project, but only works with chromium currently.

Reply:
I wonder if rather than building our own, we want to use one of the pre-built images that Microsoft publishes (https://playwright.dev/docs/docker, https://hub.docker.com/_/microsoft-playwright, https://mcr.microsoft.com/v2/playwright/tags/list, https://playwright.dev/docs/ci#docker).

> These image includes all the dependencies needed to run browsers in a Docker container, and also include the browsers themselves.

I am encouraged to see that they are pushing for both `amd64` and `arm64`... though I am not willing to believe that all three browsers work until I see it. 

Maybe `v1.17.0-focal` would do it for us, at least as a base image? I think we're not attached to debian in this context, since this won't be deployed anywhere. @bensteinberg would you agree, ubuntu is fine for a testing container?

Or are you thinking we should build our own?

Reply:
I agree, Ubuntu is fine here.